mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,38 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+from django.utils.six.moves import urllib
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        try:
+            code_type = CodeType.objects.get(code='govuk_slug')
+        except CodeType.DoesNotExist:
+            code_type = CodeType(
+                code='govuk_slug',
+                description='Slug for use by GOV.UK'
+            )
+            code_type.save()
```

Comment:
The standard code types are defined in [a fixture](https://github.com/alphagov/mapit/blob/master/mapit_gb/fixtures/uk.json#L57-L60) - is there any reason not to add this one there instead of creating it here? I guess we'd want to avoid future primary key clashes in case mySociety add any more in future, but could give this one a high `pk` instead of `5` to mitigate that.